### DISCUSSION TITLE
Where does Directus stores the language list?

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#18168] [Archive]
Searching on a language name, I only find it in index.f8ba9f9b.entry.js
But hard modifying for instance `Breton` to `Brezhoneg` does not do any change in the language list we get inside Directus.
This just to put my question on the table:

My motivation in fact is the following: I have vendors sharing the same database, and the same forms.
But they have different ways to designate the same fields.

So I'm searching how I could use the Directus translation functionality to assign a "language" to the member of a vendor that fits their designations and so, have aa-US, bb-US, ...zz-US vendor "languages", sharing the same basics but with some differences.

I searched the DB with no luck. Same with the .js files.
And obviously this is not something that would go through Crowdin.

Any hint to put me on the right track?

**Comment by hanneskuettner** [comment]
The available languages are hard coded here in [`available-languages.yaml`](https://github.com/directus/directus/blob/main/app/src/lang/available-languages.yaml)
